otx/algorithms/detection/task.py

```diff
@@ -91,21 +92,33 @@ def __init__(self, task_environment: TaskEnvironment, output_path: Optional[str]
         )
         self._anchors: Dict[str, int] = {}
 
-        if hasattr(self._hyperparams, "postprocessing") and hasattr(
-            self._hyperparams.postprocessing, "confidence_threshold"
-        ):
-            self.confidence_threshold = self._hyperparams.postprocessing.confidence_threshold
+        if hasattr(self._hyperparams, "postprocessing"):
+            if hasattr(self._hyperparams.postprocessing, "confidence_threshold"):
+                self.confidence_threshold = self._hyperparams.postprocessing.confidence_threshold
         else:
             self.confidence_threshold = 0.0
 
         if task_environment.model is not None:
             self._load_model()
 
+        self.use_ellipse_shapes = self._hyperparams.postprocessing.use_ellipse_shapes
+
         if self._hyperparams.tiling_parameters.enable_tiling:
             self.data_pipeline_path = os.path.join(self._model_dir, "tile_pipeline.py")
         else:
             self.data_pipeline_path = os.path.join(self._model_dir, "data_pipeline.py")
 
+    def _load_postprocessing(self, model_data):
```

Comment:
I think overall this function could be moved to _load_model_ckpt in one sentence.

self._hyperparams.postprocessing = model_data.get("config", {}).get("postprocessing", {})["use_ellipse_shapes"]["value"]

Of course, we can add some check statement before assignment

Reply:
Agree with your opinion, 

For further implementation, if we want to change other postprocessing values, then this function will be needed. That's the reason why I made special block for only single variable.